### PR TITLE
Removing redundant sharp sign

### PR DESCRIPTION
**Pull request by afshinm**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -7,7 +7,8 @@
 /// # Examples
 ///
 /// ```should panic
-/// # #[macro_use] extern crate assert_approx_eq;
+/// #[macro_use] 
+//  extern crate assert_approx_eq;
```
Comment (ashleygwilliams) [inline src/lib.rs]: could you make this a `///`? thanks!

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]
thank you so much for this! one little nit and then i'll merge and publish a patch version! :)

File: src/lib.rs (hunk 1 above)
Comment (afshinm) [reply src/lib.rs]: ops, sorry. sure.

**Comment by afshinm** [comment]
done 👍

File: src/lib.rs [hunk 2]
```diff
@@ -7,7 +7,8 @@
 /// # Examples
 ///
 /// ```should panic
-/// # #[macro_use] extern crate assert_approx_eq;
+/// #[macro_use] 
+///  extern crate assert_approx_eq;
```
Comment (ashleygwilliams) [inline src/lib.rs]: sorry to be nitpicky but it appears there is a space here! mind removing that as well?

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]

**Comment by afshinm** [comment]
ah sorry about that. done.

**Comment by afshinm** [comment]
ping!

**Pull request closed without merging**